Makefile

```diff
@@ -150,3 +147,16 @@ node_modules/.uptodate: package.json
 	@echo installing javascript dependencies
 	@node_modules/.bin/check-dependencies 2>/dev/null || npm install
 	@touch $@
+
+PYTHON_VERSION = 3.6.8
+PYTHON_MINOR_VERSION := $(shell echo $$PYTHON_VERSION | cut -d '.' -f1,2)
+.PHONY: python
+python:
+	@if [ `python${PYTHON_MINOR_VERSION} --version 2>/dev/null | grep -c "${PYTHON_VERSION}"` -ne 1 ]; then pyenv install -s ${PYTHON_VERSION}; fi
```

Comment:
This runs `python3.6 --version` and if it outputs anything other than 3.6.8, or fails because there's no `python3.6` executable on the path, then it runs `pyenv install`